zrml/court/src/lib.rs

```diff
@@ -187,315 +169,1166 @@ mod pallet {
         type WeightInfo: WeightInfoZeitgeist;
     }
 
-    #[pallet::error]
-    pub enum Error<T> {
-        /// It is not possible to insert a Juror that is already stored
-        JurorAlreadyExists,
-        /// An account id does not exist on the jurors storage.
-        JurorDoesNotExists,
-        /// On dispute or resolution, someone tried to pass a non-court market type
-        MarketDoesNotHaveCourtMechanism,
-        /// No-one voted on an outcome to resolve a market
-        NoVotes,
-        /// Forbids voting of unknown accounts
-        OnlyJurorsCanVote,
-    }
+    // Number of jurors for an initial market dispute
+    const INITIAL_JURORS_NUM: usize = 5;
```

Comment:
06af2fbfeac6fe208b8ae9ce408ab77a8480172d

Reply:
Like assuming people might use this pallet in the future and only change config parameters (not the code). It's safer for us internally to set the formula, so that configurators don't mess up.